src/utilities/util.h

```diff
@@ -20,10 +20,15 @@ extern bool verbose;             /* Verbose flag */
 
 extern WT_EVENT_HANDLER *verbose_handler;
 
+/* WT-specific return codes for __wt_getopt(); use __wt_optwt to enable */
+#define WT_GETOPT_BAD_OPTION 1
+#define WT_GETOPT_BAD_ARGUMENT 2
```

Comment:
Hmm, these appear again, so 1. please alphabetize but 2. please figure out if we can have these in one WT header file so that changing their values applies everywhere.

These source files use other WT_ macros. I think if you added these to `src/include/misc.h` they'd be available everywhere, and then remove from `os_getopt.c`.